sdk/simulator-service/src/strategies/refinanceAnyPair/RefinanceLendingToLendingAnyPair.ts

```diff
@@ -95,13 +87,20 @@ export async function refinanceLendingToLendingAnyPair(
       }),
       isCollateralSwapSkipped,
     )
+    .next(async () => ({
+      name: 'OpenTargetPosition',
+      type: SimulationSteps.OpenPosition,
+      inputs: {
+        pool: targetPool,
+      },
+    }))
     .next(async (ctx) => ({
-      name: 'DepositBorrowToTarget',
+      name: 'DepositBorrowToTargetPosition',
       type: SimulationSteps.DepositBorrow,
       inputs: {
         // refactor
         borrowAmount: isDebtSwapSkipped
-          ? ctx.getReference(['PaybackWithdrawFromSource', 'paybackAmount'])
+          ? ctx.getReference(['SwapCollateralFromSourcePosition', 'received'])
```

Comment:
small naming consistenty improvements are really nice adding clarity and additional context

Reply:
👏